js/tools/highlighter.js

```diff
@@ -0,0 +1,33 @@
+window.tools.highlighter = {
```

Comment:
We're getting some linter errors here because you're using a global variable (`app`) which isn't defined in this file (it's created in another file, specifically the `index.html` page where the instanceof the artware.js framework is created), we need to specify which global variables we're using in each file by adding a comment at the top like this: `/* global app */`